docs/src/components/productCore/CoreHero2.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+import Typography from '@mui/material/Typography';
+import GradientText from 'docs/src/components/typography/GradientText';
+import IconImage from 'docs/src/components/icon/IconImage';
+
+export default function HeroPricing() {
```

Comment:
The default export name should generally match the file name. Also, could we have something more descriptive than CoreHero2?

Reply:
Good catch! I've changed it to just `CoreHero` to follow the pattern used in all other marketing pages (i.e. `{product}Hero` for the hero section).